src/test/java/com/google/api/generator/gapic/model/MessageTest.java

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.model;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.assertThrows;
+
+import com.google.api.generator.engine.ast.TypeNode;
+import com.google.api.generator.engine.ast.VaporReference;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class MessageTest {
+
+  public static final String MESSAGE_NAME = "TestMessage";
+  public static final Message.Builder TEST_MESSAGE_BUILDER =
+      Message.builder()
+          .setName(MESSAGE_NAME)
+          .setFullProtoName("com.google.test.TestMessage")
+          .setType(TypeNode.OBJECT);
+
+  @Test
+  public void validateField_shouldThrowExceptionIfFieldNameIsEmpty() {
+    Message message = TEST_MESSAGE_BUILDER.build();
+    IllegalStateException illegalStateException =
+        assertThrows(
+            IllegalStateException.class, () -> message.validateField("", ImmutableMap.of()));
+    assertThat(illegalStateException.getMessage())
```

Comment:
```suggestion
    assertThat(illegalStateException).hasMessageThat()
```
I think `illegalStateException` is too verbose in this context. I prefer `assertThat(exception)`.

Reply:
Updated.